### ISSUE TITLE
ILM policy keeps getting overwritten

### ISSUE THREAD

**Issue by JohannesMahne** [elastic/beats#27270]
For confirmed bugs, please report:
- Version: 7.12.1
- Operating System: ECE
- Steps to Reproduce: 

We are seeing ILM policies that were altered via Kibana get reverted automatically. We presume this is done by Beats. We also observe extremely high version numbers on these ILM policies, indicating that these reversions are happening a lot and have been for a while.

@urso - please advise.

**Comment by urso** [comment]
Sounds like the ILM policy is overwritten whenever Beats reconnect to ES (e.g. after IO error or restart). Please share `setup.ilm.*` settings.

**Comment by hendry-lim** [comment]
We are experiencing the same issue but only with Heartbeat, even with `setup.ilm.overwrite` setting set to `false`. We are using Heartbeat `7.13.3`.
```
setup.ilm.check_exists: false
setup.ilm.overwrite: false
setup.template.type: index
```

**Comment by dsent9** [comment]
@urso 

`setup:
  kibana.host: "https://${kibana_url}:9243"
  template.settings:
    index.number_of_shards: 1
    index.number_of_replicas: 1
    index.codec: best_compression
  ilm:
    enabled: auto
    rollover_alias: "auditbeat-%{[agent.version]}"
    pattern: "{now/d}-000001"
    policy_file: ${path.config}/ilm-policy.json
`

**Comment by jlind23** [comment]
@JohannesMahne is this issue still relevant?

**Comment by kvch** [comment]
Closing this issue, as a PR has been released in 7.15 with a possible fix: https://github.com/elastic/beats/pull/27508

Also, in general if you do not want to load ILM policies everytime your Beat starts set `setup.ilm.enabled` to false.

If the issue persist, please reopen it.
